requirements.in

```diff
@@ -20,7 +20,7 @@ factory-boy
 fakeredis
 freezegun
 https://github.com/codecov/opentelem-python/archive/refs/tags/v0.0.4a1.tar.gz#egg=codecovopentelem
-https://github.com/codecov/shared/archive/ca82e9dcf56fb27f012371d6f32f70fee0a4605f.tar.gz#egg=shared
```

Comment:
to be updated to the head of main commit once https://github.com/codecov/shared/pull/275 is merged